tests/basic/storage_pool_test.go

```diff
@@ -2053,4 +2055,471 @@ var _ = Describe("{VolUpdateAddDrive}", func() {
 		defer EndTorpedoTest()
 		AfterEachTest(contexts, testrailID, runID)
 	})
+
+})
+
+var _ = Describe("{StoPoolExpMulPools}", func() {
+	/*
+		Steps : Verify if Multiple Pools exist in the system
+				Expand one of the available pools present from the list of available pools
+	*/
+	var testrailID = 51298
+	// testrailID corresponds to: https://portworx.testrail.net/index.php?/cases/view/51298
+	// testrailID Description : Having multiple pools and resize only one pool
+	var runID int
+	JustBeforeEach(func() {
+		StartTorpedoTest("StoPoolExpMulPools", "Validate storage pool expansion using resize-disk option when multiple pools are present on the cluster", nil, testrailID)
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+	})
+
+	// Schedule the applications to run on K8s cluster
+	var contexts []*scheduler.Context
+	stepLog := "Has to schedule apps, and expand it by resizing a pool "
+	It(stepLog, func() {
+		log.InfoD(stepLog)
+
+		contexts = make([]*scheduler.Context, 0)
+
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("poolexpand-%d", i))...)
+		}
+		ValidateApplications(contexts)
+
+		// Get all the storage Nodes present in the system
+		stNodes := node.GetStorageNodes()
+		if len(stNodes) == 0 {
+			dash.VerifyFatal(len(stNodes) > 0, true, "Storage nodes found!")
+		}
+		log.InfoD("All Storage Nodes present on the kubernetes cluster %s", stNodes)
+
+		/* Validate if the Node with Multiple pools are available ,
+		if, any node has multiple pools present , then use that Node for expanding
+		else, Fail the test case
+		*/
+		var selectedNode node.Node
+		isMultiPoolNode := false
+		for _, selNode := range stNodes {
+			log.InfoD("Validating Node %s for multipool configuraitons", selNode.Name)
+			if len(selNode.StoragePools) > 1 {
+				isMultiPoolNode = true
+				selectedNode = selNode
+				break
+			}
+		}
+
+		dash.VerifyFatal(isMultiPoolNode, true, "Multipool Node found ?")
+		// Selecting Storage pool based on Pools present on the Node
+		selectedPool, err := GetPoolWithIOsInGivenNode(selectedNode)
+		log.FailOnError(err, "error identifying volume")
+
+		stepLog = fmt.Sprintf("Expanding pool on node %s and pool UUID: %s using auto", selectedNode.Name, selectedPool.Uuid)
+		Step(stepLog, func() {
+			poolToBeResized, err := GetStoragePoolByUUID(selectedPool.Uuid)
+			log.FailOnError(err, "Failed to get pool using UUID ")
```

Comment:
Make this error more verbose by adding UUID
`log.FailOnError(err, "Failed to get pool using UUID [%s]", selectedPool.Uuid)`